packages/discovery-provider/scripts/start.sh

```diff
@@ -65,10 +65,10 @@ else
         [ -e /var/celerybeat.pid ] && rm /var/celerybeat.pid
         audius_service=beat celery -A src.worker.celery beat --schedule=/var/celerybeat-schedule --pidfile=/var/celerybeat.pid --loglevel WARNING 2>&1 | tee >(logger -t beat) &
         # start worker dedicated to indexing ACDC
-        audius_service=index_nethermind_worker celery -A src.worker.celery worker -Q index_nethermind --loglevel $audius_discprov_loglevel 2>&1 | tee >(logger -t index_nethermind_worker) &
+        audius_service=worker celery -A src.worker.celery worker -Q index_nethermind --loglevel $audius_discprov_loglevel --hostname=index_nethermind --concurrency 1 2>&1 | tee >(logger -t index_nethermind_worker) &
```

Comment:
this has the same service name as the other workers `worker`, do we not want the keep the name `index_nethermind_worker`?

Reply:
audius_service is just an env var used for logging and this changes it back to what it was. it's still the same service so i wanted to revert this part.

i actually added --hostname so these workers can be differentiated so it proved the memory issues were on other tasks besides index_nethermind

Reply:
cool thanks!